framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -435,9 +442,13 @@ private void parse(
                 stubWarnIfNotFound("StubParser ignoring: " + elt);
             }
         }
+        typeParameters = new ArrayList<>();
     }
 
-    private void parseType(
+    /**
+     * @return List of AnnotatedTypeMirrors of the type's type parameters
```

Comment:
ATM -> ATV.
I assume all returned ATV will be declaration type variables, not use.